### ISSUE TITLE
DAL unit test is failing

### ISSUE THREAD

**Issue by julsemaan** [inverse-inc/packetfence#2054]
This breaks the devel build

```
# perl t/unittest/dal.t 
1..30
ok 1 - Node does not exists
ok 2 - New node not the database
ok 3 - New node not in the database
ok 4 - node->voip is default 'no'
ok 5 - Saving ff:ff:ff:ff:ff:fe into the database
ok 6 - New node is in the database
ok 7 - Found node in database
ok 8 - Only saving values that changed
ok 9 - Saving changes into the database
ok 10 - Reloading node from database
ok 11 - Changes were saved into database
ok 12 - Cannot save invalid enum value into the database
ok 13 - Save valid data into the database
ok 14 - Reloading node from database
ok 15 - Changes were saved into database
ok 16 - Cannot save a null value into the database
ok 17 - Remove node in database
ok 18 - Node does not exists
ok 19 - Saving node after being deleted
ok 20 - Saving node after being deleted from under us
ok 21 - Saving after being deleted
ok 22 - Voip was saved
ok 23 - new node saved with default values
ok 24 - Save node with computername = zams-computer
ok 25 - Save node2 with voip = yes
ok 26 - Saving different values do not conflict
ok 27 - Saving different values do not conflict
ok 28 - ff:ff:ff:ff:ff:fe was successfully created
not ok 29 - ff:ff:ff:ff:ff:fe was successfully updated
#   Failed test 'ff:ff:ff:ff:ff:fe was successfully updated'
#   at t/unittest/dal.t line 153.
#          got: '201'
#     expected: '200'
ok 30 - no warnings
# Looks like you failed 1 test of 30.
```

**Comment by jrouzierinverse** [comment]
Sadly this works in my environment.

**Comment by julsemaan** [comment]
It fails on pfbuilder and in my environment so please advise on what adjustment could be necessary

**Comment by jrouzierinverse** [comment]
FIxed in devel by 5bcae9af6667480a0b811e4e75b10882905dd982 reopen if you found to be an issue
